cmd/fleet/main.go

```diff
@@ -81,6 +81,15 @@ func applyDevFlags(cfg *config.FleetConfig) {
 		SecretAccessKey:  "minio123!",
 		DisableSSL:       true,
 		ForceS3PathStyle: true,
```

Comment:
Should those use the new `carves_`-prefixed fields?

Reply:
Good catch! Missed this because the first implementation kept the same field names 😅